src/components/macroview/rightPanel/editForms/MousePressForm.tsx

```diff
@@ -52,15 +92,28 @@ export default function MousePressForm({
     if (selectedElement.data.data.type !== 'DownUp') {
       return
     }
-    let duration
-    if (mousepressDuration === '') {
-      duration = 0;
-    } else {
+    let duration = 20
 
-      duration = parseInt(mousepressDuration)
-      if (Number.isNaN(duration)) {
-        return
-      }
+    if (Number(mousepressDuration) >= 20) {
+      duration = Number(mousepressDuration)
+    } else if (mousepressDuration === '') {
+      toast({
+        title: 'Default duration applied',
+        description: 'Applied default duration of 20ms',
+        status: 'info',
+        duration: 4000,
+        isClosable: true
+      })
+    } else if (Number(mousepressDuration) < 20) {
+      toast({
+        title: 'Minimum duration',
+        description: 'Duration must be at least 20ms',
+        status: 'warning',
+        duration: 4000,
+        isClosable: true
+      })
+    } else if (Number.isNaN(duration)) {
```

Comment:
Please use a constant for this magic number. The amount of unique hardcoded references to 20 is making me cry. Just imagine how annoying it will be if you ever want to change this number

Reply:
I think i changed it in a lot of places but forgot about a few, will change

Reply:
Tweaked: [5d256ec](https://github.com/WootingKb/wooting-macros/pull/194/commits/5d256ecf2bf5fb0a8a4c1027963d6ec4b0fe7aa8)